recipe/meta.yaml

```diff
@@ -12,15 +12,15 @@ source:
   sha256: 2e6e113aa02aee4a8de635900307a87a37d052d2f12427df71b101ef403ce7b1
 
 build:
-  number: 0
+  number: 1
   script:
     - go install -v -ldflags "-X main.VERSION={{ version }}" .
 
 requirements:
   build:
     - {{ compiler('go') }}
   run:
-    - terraform 0.12
+    - terraform 0.13
```

Comment:
I just noticed terragrunt 0.27 should also work with terraform 0.14 per the release notes: https://github.com/gruntwork-io/terragrunt/releases/tag/v0.27.0

Should this be a range or unpinned and let people rely on eg. https://terragrunt.gruntwork.io/docs/reference/config-blocks-and-attributes/#terraform_version_constraint to set their desired terraform version?

Reply:
I've opted for a range of terraform versions with known support rather than unpinned, as it seems like the safer choice.